.github/workflows/gradle_tests.yaml

```diff
@@ -0,0 +1,24 @@
+name: Java Gradle Tests
+on:
+  push:
+    branches:
+      - master
+  #maybe even on PR-s?
+
+jobs:
+  build:
+    runs-on: ubuntu-latest #?
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: JDK setup
+        uses: actions/setup-java@v2
+        with:
+          java-version: 'latest' #or any specific version?
```

Comment:
A latest helyett jobb lenne egy konkrét verziót használni és majd manuálisan frissíteni, mert új java verziókban lehetnek apróbb változtatások, amik problémát okozhatnak. Felőlem lehet a java 22 is, de @koczoa java verzióját lehet jobb lenne használni, mert az ő gépén kell biztosan futnia.

Reply:
@koczoa milyen java verzió van a gépeden?

Reply:
Legyen java 17 csak hogy le tudjuk zárni a PR-t. Ha kell valami egy újabb verzióból, akkor majd később növeljük.

Reply:
java 17, legyen az, biztos, ami biztos